### PR TITLE
Use separate class for App Center Distribute module for play store target builds

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR creates separate target to be used in Play Store targets.

[AB#81916](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/81916)

### CODE REVIEW

File: SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.AppCenter.Test.Functional")]
```
Comment (MatkovIvan) [inline SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Properties/AssemblyInfo.cs:6]: Do you use it in functional tests?

Comment (Jamminroot) [reply SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Properties/AssemblyInfo.cs:6]: Nope. Not fixed yet

File: SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <Import Project="..\Microsoft.AppCenter.Distribute.Shared\Microsoft.AppCenter.Distribute.Shared.projitems" Label="Shared" />
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
```
Comment (MatkovIvan) [inline SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj:5]: Let's target it directly to Android

File: nuget/AppCenterDistributePlay.nuspec [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <iconUrl>https://aka.ms/k76877</iconUrl>
+    <dependencies>
+      <dependency id="Microsoft.AppCenter" version="$version$" />
+    </dependencies>
+    <version>$version$</version>
+  </metadata>
+  <files>
+
+    <!-- Android -->
+    <file src="$android_dir$/Microsoft.AppCenter.DistributePlay.dll" target="lib/MonoAndroid403" />
```
Comment (MatkovIvan) [inline nuget/AppCenterDistributePlay.nuspec:24]: dll must have the same name. Remove postfix

File: SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <Import Project="..\Microsoft.AppCenter.Distribute.Shared\Microsoft.AppCenter.Distribute.Shared.projitems" Label="Shared" />
+  <PropertyGroup>
+    <TargetFramework>monoandroid70</TargetFramework>
```
Comment (MatkovIvan) [inline SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj:5]: We already have empty implementation for .NET Standard, please reuse that sources and project file

File: .github/workflows/codeql.yml [hunk 5]
```diff
@@ -102,6 +102,7 @@ jobs:
         MSBuild.exe SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Android/Microsoft.AppCenter.Analytics.Android.csproj -property:UseSharedCompilation=false -property:Configuration=Release /restore -target:Build
         MSBuild.exe SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Microsoft.AppCenter.Crashes.Android.csproj -property:UseSharedCompilation=false -property:Configuration=Release /restore -target:Build
         MSBuild.exe SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Microsoft.AppCenter.Distribute.Android.csproj -property:UseSharedCompilation=false -property:Configuration=Release /restore -target:Build
+        MSBuild.exe SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay/Microsoft.AppCenter.DistributePlay.csproj -property:UseSharedCompilation=false -property:Configuration=Release /restore -target:Build
```
Comment (AnastasiaKubova) [inline .github/workflows/codeql.yml:105]: I think that it already unnecessary

File: SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute/Microsoft.AppCenter.Distribute.csproj [hunk 6]
```diff
@@ -2,7 +2,7 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <Import Project="..\Microsoft.AppCenter.Distribute.Shared\Microsoft.AppCenter.Distribute.Shared.projitems" Label="Shared" />
   <PropertyGroup>
-    <TargetFramework>netstandard1.0</TargetFramework>
+    <TargetFrameworks>netstandard1.0;MonoAndroid70</TargetFrameworks>
```
Comment (AnastasiaKubova) [inline SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute/Microsoft.AppCenter.Distribute.csproj:5]: I think we need to add the string resources to this project which used in the app updates dialogs

File: .github/workflows/codeql.yml (hunk 5 above)
Comment (Jamminroot) [reply .github/workflows/codeql.yml:105]: Good point, forgot about this one

File: scripts/configuration/ac-build-config.xml [hunk 7]
```diff
@@ -63,6 +66,7 @@
     <module dotnetModule="SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes" nuspec="AppCenterCrashes.nuspec"/>
     <module dotnetModule="SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute" nuspec="AppCenterDistribute.nuspec"/>
     <module dotnetModule="SDK/AppCenterPush/Microsoft.AppCenter.Push" nuspec="AppCenterPush.nuspec"/>
+    <module dotnetModule="SDK/AppCenterDistribute/Microsoft.AppCenter.DistributePlay" nuspec="AppCenterDistributePlay.nuspec"/>
```
Comment (MatkovIvan) [inline scripts/configuration/ac-build-config.xml:69]: Sort

**Pull request merged**